website/content/docs/auth/azure.mdx

```diff
@@ -390,3 +395,73 @@ namespace Examples
 </CodeBlockConfig>
 
 </CodeTabs>
+  
+## Authentication Flows and Role Restrictions
+  
+This section provides more details about the authentication flows and associated restrictions that be supplied in the the role definition.
+  
+There are 3 core flows and associated methods of restricting a role in the Azure Auth Method:
+- JWT Flow: Identity based restrictions that use claims from the JWT.
+- VM Flow: Virtual Machine (VM) based restrictions that use data from the Virtual Machine.
+- VMSS Flow: Virtual Machine Scale Set (VMSS) based restrictions that use data from the Virtual Machine Scale Set.
+  
+The VM and VMSS based restrictions can only be used for those resource types. If using Azure Kubernetes Service, Azure Container Instance or some other resource type that supports a managed identity, you cannot apply the restrictions applicable to VM and VMSS.
+  
+The restrictions that apply only to VM or VMSS are:
+- Locations `bound_locations`.
+- Subscriptions `bound_subscription_ids`.
+- Resource groups `bound_resource_groups`.
+- Scale sets `bound_scale_sets`. (VMSS only).
+  
+The restrictions that apply to any resource type come from the JWT and these are:
```

Comment:
```suggestion
The restrictions that apply to any resource types that comes from the JWT and these are:
```

Reply:
Made a different change to this to make it better.